features-json/custom-elementsv1.json

```diff
@@ -0,0 +1,281 @@
+{
+  "title":"Custom Elements v1",
+  "description":"Method of defining new HTML tags.",
+  "spec":"https://html.spec.whatwg.org/multipage/scripting.html#custom-elements",
+  "status":"ls",
+  "links":[
+    {
+      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=889230",
+      "title":"Firefox tracking bug: Implement Custom Elements (from Web Components)"
+    },
+    {
+      "url":"http://status.modern.ie/customelements",
```

Comment:
This was a copy from `customelements.json` but happy to remove if that's the case